buildLogAnalyzer.py

```diff
@@ -123,6 +123,8 @@ def __init__(
             "ok": "ok",
         }
 
+        self.cmsswVersion = os.getenv("CMSSW_VERSION", "master")
```

Comment:
@iarspider , any specific reason to not use `self.release`?

Reply:
what about this?

Reply:
I misread your comment. Sure, no reason.